src/web/src/components/SearchHelp.tsx

```diff
@@ -1,72 +1,82 @@
-import { withStyles, makeStyles } from '@material-ui/core/styles';
-import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Theme, useTheme } from '@material-ui/core';
 
-import { Typography, Tooltip } from '@material-ui/core';
-
-const useStyles = makeStyles(() => ({
-  infoButton: {
-    marginTop: '7.5px',
-    fontSize: '3rem',
-    color: '#8BC2EB',
+const useStyles = makeStyles((theme: Theme) => ({
+  container: {
+    width: '50%',
+    transform: 'translate(-50%, -50%)',
+    position: 'absolute',
```

Comment:
I am not a CSS God, but I really don't like position: absolute. 
Is there any reasoning why using it here? 
This comment is not a request, more of a comment - I'd personally do it via Flexbox if possible.

Reply:
just another way to align the container itself to the centre of its parent, it's okay

Reply:
okay, this can be resolved than